docs/nodes/AI_ML/CLASSIFICATION/ONE_HOT_ENCODING/examples/EX1/example.md

```diff
@@ -1,3 +1,3 @@
-In this example, `ONE_HOT_ENCODING` is passed the `tips` dataset from the `PLOTLY_DATASET` node.
+In this example, the `tips` dataset from the `PLOTLY_DATASET` node is passed to `ONE_HOT_ENCODING`.
```

Comment:
Should we mention "node" when referring to nodes? I saw that elsewhere we referred to "`EXTRACT_COLUMNS` node". Should we do this every time? So this would be "...passed to the `ONE_HOT_ENCODING` node".